torch/_dynamo/eval_frame.py

```diff
@@ -666,8 +666,13 @@ def result_capturing_wrapper(*graph_inputs):
 
             graph_captured_input = graph_inputs
             assert graph is not None
-            graph_captured_result = graph(*graph_inputs)
-            return graph_captured_result
+            if tracing_mode != "symbolic":
+                graph_captured_result = graph(*graph_inputs)
+                return graph_captured_result
+            for node in graph.graph.nodes:
+                if node.op == "output":
+                    # If we are in symbolic, does it matter what we return?
+                    return tuple([torch.zeros([0]) for t in node._args])
```

Comment:
@jansel is this sound? Or do we need to follow output spec here too?

Reply:
Why do we need to do something different with symbolic versus not symbolic here?

Reply:
context: https://fb.workplace.com/groups/1075192433118967/permalink/1205760686728807/

Basically, in symbolic mode, we cannot always run the graph with real inputs, because folks use symbolic mode to get symbolic trace, and make_fx with symbolic never runs the kernels (I just learned this in that post). So, to preserve the symbolic meaning here, and support that usecase, we avoid running the actual kernels, graph, ops, etc - in favor of deferring until make_fx.

Reply:
I think you should stop abusing the standard torchdynamo compile interface, and just call into the symbolic evaluator directly. That will solve most of the question here

Reply:
Hmm, just bypass dynamo?